### PR TITLE
Prepare for December release

### PR DESCRIPTION
**Pull request by DonJayamanne**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,72 @@
...
+   ([#340](https://github.com/Microsoft/vscode-jupyter/issues/340))
+1. Allow user to cancel asking about logging level.
+   ([#348](https://github.com/Microsoft/vscode-jupyter/issues/348))
+1. Watch for any addition of the python extension, and don't suggest a full reload when it is added.
+   ([#405](https://github.com/Microsoft/vscode-jupyter/issues/405))
+1. Only offer to export to python script when the metadata specifies python as its language.
+   ([#407](https://github.com/Microsoft/vscode-jupyter/issues/407))
+1. Hide webview based Notebook command `Select Kernel` when a Notebook is opened using the new VS Code Native Notebook editor.
+   ([#426](https://github.com/Microsoft/vscode-jupyter/issues/426))
+1. CTRL+1/CTRL+2 switching with the interactive window does not put the focus back in the edit box.
```
Comment (rchiodo) [inline CHANGELOG.md:20]: This one isn't technically fixed yet. Still waiting on VS code to take my PR.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,72 @@
...
+   ([#3991](https://github.com/Microsoft/vscode-jupyter/issues/3991))
+1. Fix variable fetching on remote machines that don't have our scripts files on them.
+   ([#4006](https://github.com/Microsoft/vscode-jupyter/issues/4006))
+1. Display survey prompt once per session.
+   ([#4077](https://github.com/Microsoft/vscode-jupyter/issues/4077))
+1. Guard against AttributeErrors in our DataViewer code.
+   ([#4082](https://github.com/Microsoft/vscode-jupyter/issues/4082))
+1. Ensure user cannot belong to Custom Editor experiment is already in Native Notebook experiment.
+   ([#4105](https://github.com/Microsoft/vscode-jupyter/issues/4105))
+
```
Comment (rchiodo) [inline CHANGELOG.md:38]: You seem to be missing the news entry I added on my last submission

Comment (rchiodo) [reply CHANGELOG.md:38]: 1701.md

**Review by rchiodo — CHANGES_REQUESTED** [review]
⏲️

**Review by rchiodo — APPROVED** [review]

**Review by joyceerhl — APPROVED** [review]

**Pull request merged**